qtpy/tests/test_qtlocation.py

```diff
@@ -16,7 +18,7 @@ def test_qtlocation():
     assert QtLocation.QGeoRoutingManager is not None
     assert QtLocation.QGeoRoutingManagerEngine is not None
     assert QtLocation.QGeoServiceProvider is not None
-    #assert QtLocation.QGeoServiceProviderFactory is not None
+    # assert QtLocation.QGeoServiceProviderFactory is not None
```

Comment:
I think `QGeoServiceProviderFactory` is not availale on PyQt5. Maybe we should removed this assert or add a validation here preventing running the assert if testing with PYQT5?

Reply:
For the tests I added, I just looked at the documentation for module or imported it in a repl and checked `dir()` to see what things were defined. If there were inconsistencies I just picked a common subset to add asserts for. 

I don't have a good sense of how specific a module import test should be, especially since `qtpy` is aimed at 6(ish) targets, and half of them are actively moving.

I'll apply these suggestions to the commented asserts, but I also think there are some issues with the current testing strategy. I'll try to organize my thoughts on this and open an issue to get some discussion going.